test/e2e/suite/serving/cainjector.go

```diff
@@ -412,46 +430,38 @@ var _ = framework.CertManagerDescribe("CA Injector", func() {
 		},
 		getCAs: func(obj runtime.Object) [][]byte {
 			crd := obj.(*apiext.CustomResourceDefinition)
-			if crd.Spec.Conversion == nil || crd.Spec.Conversion.WebhookClientConfig == nil {
+			if crd.Spec.Conversion == nil || crd.Spec.Conversion.Webhook == nil || crd.Spec.Conversion.Webhook.ClientConfig == nil {
 				return nil
 			}
-			return [][]byte{crd.Spec.Conversion.WebhookClientConfig.CABundle}
+			return [][]byte{crd.Spec.Conversion.Webhook.ClientConfig.CABundle}
 		},
 		disabled: "ConversionWebhook feature not yet enabled on test infra",
 	})
 
-	// TODO: re-enable this test.
-	// This test has been disabled in order to reduce flakes on e2e tests
-	// (hitting around 70% failure rate). This is because when running this test
-	// will cause tiller install to fail, therefore making other tests to fail
-	// when running bother at the same time. We should find a way to make this test run in serial.
-	// https://github.com/jetstack/cert-manager/issues/2353
-	// https://github.com/jetstack/cert-manager/issues/2354
-	//injectorContext("api service", &injectableTest{
-	//	makeInjectable: func(namePrefix string) runtime.Object {
-	//		return &apireg.APIService{
-	//			ObjectMeta: metav1.ObjectMeta{
-	//				Name: "corev1." + namePrefix + ".testing.cert-manager.io",
-	//				Annotations: map[string]string{
-	//					certmanager.WantInjectAnnotation: types.NamespacedName{Name: "serving-certs", Namespace: f.Namespace.Name}.String(),
-	//				},
-	//			},
-	//			Spec: apireg.APIServiceSpec{
-	//				Service: &apireg.ServiceReference{
-	//					Name:      "does-not-exit",
-	//					Namespace: "default",
-	//				},
-	//				Group:                namePrefix + ".testing.cert-manager.io",
-	//				Version:              "v1",
-	//				GroupPriorityMinimum: 1,
-	//				VersionPriority:      1,
-	//			},
-	//		}
-	//	},
-	//	getCAs: func(obj runtime.Object) [][]byte {
-	//		apiSvc := obj.(*apireg.APIService)
-	//		return [][]byte{apiSvc.Spec.CABundle}
-	//	},
-	//})
-
+	injectorContext("api service", &injectableTest{
+		makeInjectable: func(namePrefix string) runtime.Object {
+			return &apireg.APIService{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "v1." + namePrefix + ".testing.cert-manager.io",
```

Comment:
I had to modify the name of the resource to satisfy the v1 validation which required `spec.version + "." + spec.group + "." + ...`